include/dpp/cluster.h

```diff
@@ -1435,9 +1435,11 @@ class CoreExport cluster {
 	 * @brief Get all guild members
 	 *
 	 * @param guild_id Guild ID to get all members for
+	 * @param limit max number of members to return (1-1000)
+	 * @param after the highest user id in the previous page
 	 * @param callback Function to call when the API call completes.
 	 */
-	void guild_get_members(snowflake guild_id, command_completion_event_t callback);
+	void guild_get_members(snowflake guild_id, uint16_t limit, snowflake after, command_completion_event_t callback);
```

Comment:
these should be optional yes? (so, after the callback, ugh)

Reply:
Follows the scheme of `messages_get` I think.